andymond/js_testing_challenge
# UI tests

* Add test to verify nav bar presence
* Add tests to check for a few links
* Add test to check the contents of two ticker items
